misc/cf-support

```diff
@@ -380,6 +380,11 @@ else
 fi
 echo "Captured output of $syslog_cmd filtered for cf-|CFEngine"
 
+# cf- component related SELinux denials
+if command -v ausearch >/dev/null; then
+  ausearch -m avc -c cf- > "$tmpdir"/ausearch-cfengine-avcs.log
+fi
```

Comment:
This might not match all the cases, for example `httpd` or `postgres` don't start with `cf-`. And we may run many other commands that can hit issues. I think a better filter would be on the source context.

Reply:
right! I will adjust.